make test not working locally

### Description of the problem

I followed all the instruction on https://mne.tools/stable/install/contributing.html. Here: https://mne.tools/stable/install/contributing.html#running-the-test-suite, I am not able to run `make test`

### Steps to reproduce

```Python
run "make test" from the mne-python root folder.
```


### Link to data

_No response_

### Expected results

the tests would run

### Actual results

I get:

`make: *** No rule to make target `test'.  Stop.`

### Additional information

Platform             macOS-13.3.1-arm64-arm-64bit
Python               3.11.4 | packaged by conda-forge | (main, Jun 10 2023, 18:08:41) [Clang 15.0.7 ]
Executable           /Users/chapon01/Programs/miniconda3/envs/mnedev/bin/python
CPU                  arm (12 cores)
Memory               32.0 GB

Core
├☑ mne               1.5.0.dev45+gb92c88d70
├☑ numpy             1.24.4 (OpenBLAS 0.3.23 with 12 threads)
├☑ scipy             1.11.1
├☑ matplotlib        3.7.1 (backend=MacOSX)
├☑ pooch             1.7.0
└☑ jinja2            3.1.2

Numerical (optional)
├☑ sklearn           1.3.0
├☑ numba             0.57.1
├☑ nibabel           5.0.1
├☑ nilearn           0.10.1
├☑ dipy              1.7.0
├☑ openmeeg          2.5.6
├☑ pandas            2.0.3
└☐ unavailable       cupy

Visualization (optional)
├☑ pyvista           0.39.1 (OpenGL 4.1 Metal - 83.1 via Apple M2 Pro)
├☑ pyvistaqt         0.0.0
├☑ ipyvtklink        0.2.2
├☑ vtk               9.2.6
├☑ qtpy              2.3.1 (PyQt5=5.15.6)
├☑ ipympl            0.9.3
├☑ pyqtgraph         0.13.3
└☑ mne-qt-browser    0.5.1

Ecosystem (optional)
├☑ mne-bids          0.13.dev0
├☑ mne-connectivity  0.6.0dev0
└☐ unavailable       mne-nirs, mne-features, mne-icalabel, mne-bids-pipeline

--- a/Makefile
+++ b/Makefile
@@ -33,8 +33,6 @@ sample_data:
 testing_data:
 	@python -c "import mne; mne.datasets.testing.data_path(verbose=True);"
 
-pytest: test
-
 test-no-network: in
 	sudo unshare -n -- sh -c 'MNE_SKIP_NETWORK_TESTS=1 py.test mne'
 

--- a/doc/install/advanced.rst
+++ b/doc/install/advanced.rst
@@ -209,6 +209,14 @@ or by doing
 :func:`mne.viz.set_3d_options(antialias=False) <mne.viz.set_3d_options>` within
 a given Python session.
 
+Some hardware-accelerated graphics on linux (e.g., some Intel graphics cards)
+provide an insufficient implementation of OpenGL, and in those cases it can help to
+force software rendering instead with something like:
+
+.. code-block:: console
+
+    $ export LIBGL_ALWAYS_SOFTWARE=true
+
 Another issue that may come up is that the MESA software itself may be out of date
 in certain operating systems, for example CentOS. This may lead to incomplete
 rendering of some 3D plots. A solution is described in this `Github comment <https://github.com/mne-tools/mne-python/issues/7977#issuecomment-729921035>`_.

--- a/doc/install/contributing.rst
+++ b/doc/install/contributing.rst
@@ -872,7 +872,7 @@ Running the test suite
     The ``--pdb`` flag will automatically start the python debugger upon test
     failure.
 
-The full test suite can be run by calling ``make test`` from the
+The full test suite can be run by calling ``pytest -m "not ultraslowtest" mne`` from the
 ``mne-python`` root folder. Testing the entire module can be quite
 slow, however, so to run individual tests while working on a new feature, you
 can run the following line::

